python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -505,7 +505,9 @@ def load_low_bit(cls,
 
         if bigdl_lcmu_enabled:
             with ContextManagers(init_contexts):
-                kwargs["device"] = "meta"
+                if config.architectures is not None \
```

Comment:
how about adding comments on why chatglm behaves differently? and It is better to make sure the failed nightly test passed on your local machine.